recipe/meta.yaml

```diff
@@ -70,7 +72,7 @@ outputs:
         - run_cmake_cuda_tests.sh  # [linux]
       commands:
         - nvcc --version
-        - $CXX --verbose ${CXXFLAGS} -lcuda -lcudart_static test.cpp
+        - $CXX --verbose ${CXXFLAGS} -lcuda -lcudart_static test.cpp  # [linux]
```

Comment:
Could we add a similar test for Windows?

Reply:
I was looking at clang, which has a similar test on windows here: https://github.com/conda-forge/clang-win-activation-feedstock/blob/main/recipe/run_test_clang.bat
Although it also sets `CXX` to itself: https://github.com/conda-forge/clang-win-activation-feedstock/blob/main/recipe/activate-clang_win-64.bat#L3

So maybe the similar test to add is just compiling `test.cpp` with nvcc?

Reply:
Think we can use `cl`

Reply:
Maybe we can borrow and tweak [this test]( https://github.com/conda-forge/openlibm-feedstock/blob/e1a6f1ac9c7defd436125fac941b6e646b0c69a7/recipe/meta.yaml#L40 )